backends/credhub/src/test/kotlin/org/cloudfoundry/credhub/controllers/v1/certificates/CertificatesControllerTest.kt

```diff
@@ -301,6 +313,11 @@ class CertificatesControllerTest {
                                     "Set the key length for the regenerated certificate. If not provided, the key length will be the same as the original certificate.",
                                 ).type(JsonFieldType.NUMBER)
                                 .optional(),
+                            fieldWithPath("duration")
+                                .description(
+                                    "Set the duration for the regenerated certificate. If not provided, the duration will be the same as the original certificate. If smaller than minimum duration, minimum duration will get used",
```

Comment:
The "If smaller than minimum duration, minimum duration will get used" language here can be omitted, as the same topic will be discussed in the note section immediately below. See the current rendered doc: https://docs.cloudfoundry.org/api/credhub/version/main/#_generate_a_certificate_credential

So can remove from here. Instead, reword the note to reflect the changes in this PR. Aka edit here: https://github.com/cloudfoundry/credhub/blob/b1e236c790435846505791ca93bfe402c430d5c6/backends/credhub/src/docs/asciidoc/snippets/certificates-v1.adoc?plain=1#L41